api-client/src/utils/__tests__/GeoJson.spec.ts

```diff
@@ -0,0 +1,52 @@
+import LocationModel from '../../models/LocationModel'
+import { convertToPoint, embedInCollection } from '../geoJson'
+import { Feature, FeatureCollection } from 'geojson'
+
+describe('geoJson', () => {
+  const expectedGeoJsonMarkerFeature: Feature = {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [31.133859, 29.979848]
+    },
+    properties: {
+      name: 'Test'
+    }
+  }
+  describe('convertToPoint', () => {
+    it('should be transformed to GeoJson type', () => {
+      const location = new LocationModel({
+        name: 'Test',
+        address: 'Wertachstr. 29',
+        town: 'Augsburg',
+        state: 'Bayern',
+        postcode: '86353',
+        region: 'Schwaben',
+        latitude: '29.979848',
+        longitude: '31.133859',
+        country: 'DE'
+      })
```

Comment:
Maybe you could add a LocationModelBuilder similar to the `PoiModelBilder` in api-client/src/testing

Reply:
Maybe we should do that in the future if more tests are added for LocationModel